### PR TITLE
docs: Move docs from assets to storybook

### PR DESCRIPTION
**Pull request by lavanya-bmw**
## Description

Moved the docs from assets to the docs/storybook directory and add spinner documentation

## Why

Currently shared-component docs is mixed inside the asset repo and storybook.

## Issue

Ref: CPLP-3340, 3069

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

File: docs/static/button-variations.png
Comment (oyo) [inline docs/static/button-variations.png:1]: Why are all the variations almoste rectange shaped when all of our buttons have more rounded corners - or is there a design change upcoming?

**Review by oyo — COMMENTED** [review]
see question about button variations

File: docs/static/button-variations.png
Comment (lavanya-bmw) [reply docs/static/button-variations.png:1]: @jjeroch I have added assets repository images, do I need to update images with our designs?

**Review by oyo — APPROVED** [review]

File: docs/static/button-variations.png
Comment (oyo) [reply docs/static/button-variations.png:1]: Since this is not a major issue I mark this as resolved for now. If necessary we can update that image later.

**Pull request merged**